store/docs/soap-admin.txt

```diff
@@ -3168,3 +3168,34 @@ Corresponding API for outgoing/sent email filters (message structure same as abo
 </ModifyOutgoingFilterRulesRequest>
 
 <ModifyOutgoingFilterRulesResponse/>
+------------------------------
+ContactBackup API
+------------------------------
+<ContactBackupRequest <op="start|stop">>
```

Comment:
Request should be in this format <ContactBackupRequest op="start|stop" />

Reply:
updated soap-admin.txt